errext/format.go

```diff
@@ -0,0 +1,28 @@
+package errext
+
+import (
+	"errors"
+)
+
+// Format formats the given error and writes it to l.
```

Comment:
Could it be that this description of what the function does is somewhat outdated? I don't see any `l` variable in this function 😸

Reply:
Great catch! Fixed! 🙏🏻